src/app/services/actions/query-action-creators.ts

```diff
@@ -1,21 +1,25 @@
 import { writeData } from '../../../store/cache';
 import { IHistoryItem } from '../../../types/history';
 import { IQuery } from '../../../types/query-runner';
+import { DEFAULT_USER_SCOPES } from '../graph-constants';
 import { queryResponseError } from './error-action-creator';
+import { fetchScopes } from './permissions-action-creator';
 import {
   anonymousRequest, authenticatedRequest,
   isImageResponse, parseResponse, queryResponse
 } from './query-action-creator-util';
 import { addHistoryItem } from './request-history-action-creators';
 
+const scopes: string[] = DEFAULT_USER_SCOPES.split(' ');
+
 export function runQuery(query: IQuery): Function {
   return (dispatch: Function, getState: Function) => {
     const tokenPresent = getState().authToken;
     const respHeaders: any = {};
     const createdAt = new Date().toISOString();
 
     if (tokenPresent) {
-      return authenticatedRequest(dispatch, query).then(async (response: Response) => {
+      return authenticatedRequest(dispatch, query, scopes).then(async (response: Response) => {
         await processResponse(response, respHeaders, dispatch, createdAt);
```

Comment:
The declaration of scopes here is unnecessary since the implementation of authenticatedRequest places scopes there by default if non are passed.